clients/githubrepo/branches.go

```diff
@@ -411,12 +432,12 @@ func getBranchRefFrom(data *branch, rules []*repoRuleSet) *clients.BranchRef {
 	case data.BranchProtectionRule != nil:
 		rule := data.BranchProtectionRule
 
-		// Admin settings.
-		copyAdminSettings(rule, branchRule)
-
 		// Non-admin settings.
 		copyNonAdminSettings(rule, branchRule)
 
+		// Admin settings.
+		copyAdminSettings(rule, branchRule)
+
 	// Only non-admin settings are available.
```

Comment:
this is the sort of fragile ordering I was talking about. Ideally this change in order wouldn't be necessary

Reply:
it shouldn't matter. so feel free to revert or keep it. I was using it as an example to highlight the other comment, by pointing out that you had to switch the order to work. Assuming the other issue is fixed, feel free to resolve this as needed.